jsonnet/telemeter/client/kubernetes.libsonnet

```diff
@@ -244,6 +244,22 @@ local securePort = 8443;
                 record: 'cluster:telemetry_selected_series:count',
                 expr: 'max(federate_samples - federate_filtered_samples)',
               },
+              {
+                expr: ||| 
+                  sum by (job,namespace,pod) (
+                    rate(federate_requests_failed_total{job="telemeter-client"}[5m])
+                  ) > 0
+                |||,
+                labels: {
+                  severity: 'warning',
+                },
+                annotation: {
+                  description: 'Telemeter client in {{ $labels.namespace }}/{{ $labels.pod }} is failing to push metrics to telemeter server',
```

Comment:
Runbook URI or inline next-step suggestions?  Something that points folks at the relevent Events and/or logs would probably be the minimum needed to diagnose and fix whatever is going on.

Reply:
This might be better suited to add downstream. We have openshift/runbooks and could probably outline more context to possible root causes, which don't apply to upstream necessarily.

Reply:
I wouldn't mind to add more details here since it's openshift/telemeter isn't really upstream anyway.

Reply:
Added a command to get the logs and a couple of helpful pointers, let me know if it's enough